using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -1748,35 +1752,119 @@ This is just an implementation example. For more information about IDFA and how
 
 ```objectivec
 #import <AdSupport/ASIdentifierManager.h>
+#import <AppTrackingTransparency/ATTrackingManager.h>
 ...
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+-   ...
+-   
+    if (@available(iOS 14, *)) {
+        [self setAdvertisingIdentitiferUsingTrackingManager];
```

Comment:
What do you think about breaking this into two sections/two code samples:
- usage example before iOS 14
- usage example after iOS 14
Same comment for swift

Reply:
That's a good option too. I'm wondering however, if that will cause confusion as it may imply one example or the other is needed, when both implementations are required unless an app is only targeting iOS 14+.

Reply:
I'm in favor of having this be one section, as customers will be required to support iOS 10-13.x and iOS 14+, thus needing both implementations.